docs/vmtype.md

```diff
@@ -0,0 +1,34 @@
+# vmType
+
+Lima supports two ways of running guest machines:
+- [qemu](#qemu)
+- [vz](#vz)
+
+## QEMU
+"qemu" option makes use of QEMU to run guest operating system. 
+This option is used by default if "vmType" is not set.
+
+## VZ
+"vz" option makes use of native virtualization support provided by macOS Virtualization.Framework.
+
+An example configuration:
+```yaml
+# Example to run ubuntu using vmType: vz instead of qemu (Default)
+vmType: "vz"
+images:
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+mounts:
+  - location: "~"
```

Comment:
Can we specify virtiofs here?
We can also consider making it default for vz, but it can be another PR if controversial.

Reply:
We can will add now.

Reply:
Thanks, yes, changing the default can be another PR